dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -172,11 +173,30 @@ const containerStylesUntilLeftCol = css`
 	}
 `;
 
+const containerScrollableStylesFromLeftCol = css`
+	${between.leftCol.and.wide} {
+		grid-template-rows:
+			[headline-start controls-start] auto
+			[controls-end content-toggleable-start content-start] auto
+			[headline-end treats-start] auto
+			[content-end content-toggleable-end treats-end bottom-content-start] auto
+			[bottom-content-end];
+	}
+
+	${from.wide} {
+		grid-template-rows:
+			[headline-start content-start content-toggleable-start controls-start] auto
+			[headline-end treats-start] auto
+			[content-end content-toggleable-end treats-end controls-end bottom-content-start] auto
+			[ bottom-content-end];
```

Comment:
I presume the spacing after the `[` in `[ bottom-content-end]` isn't an issue, but I never know with CSS.